src/Miso/Internal.hs

```diff
@@ -175,24 +175,26 @@ sample _ = do
   where
     name = ms $ symbolVal (Proxy @name)
 -----------------------------------------------------------------------------
-sample_
+-- | Like @sample@ except used for dynamic @Component@ where the component-id
+-- has been retrieved via @ask@.
+sample'
```

Comment:
You might want to document the argument: 
:: MisoString -- ^ Id of the component whose model we want to sample